service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/cmo/CMOReadyToSealEventHandlerTest.java

```diff
@@ -0,0 +1,167 @@
+package uk.gov.hmcts.reform.fpl.handlers.cmo;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.events.cmo.CMOReadyToSealEvent;
+import uk.gov.hmcts.reform.fpl.handlers.HmctsAdminNotificationHandler;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.Judge;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
+import uk.gov.hmcts.reform.fpl.model.event.EventData;
+import uk.gov.hmcts.reform.fpl.model.notify.cmo.CMOReadyToSealTemplate;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.cmo.CMOReadyToSealContentProvider;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE_JUDGE;
+import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.HIS_HONOUR_JUDGE;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+@ExtendWith(MockitoExtension.class)
+class CMOReadyToSealEventHandlerTest {
+
+    private static final String HMCTS_ADMIN_EMAIL = "admin@hmcts.gov.uk";
+    private static final String HMCTS_JUDGE_EMAIL = "judge@hmcts.gov.uk";
+
+    @Mock
+    private NotificationService notificationService;
+
+    @Mock
+    private HmctsAdminNotificationHandler adminNotificationHandler;
+
+    @Mock
+    private CMOReadyToSealContentProvider contentProvider;
```

Comment:
I think NotificationServicce is the only really one thing I would mock. This might be unpopular opinion, but I don't like religious "unit" tests and mocking every possible dependency - it makes tests like that extremely brittle while refactoring. Would you expect the code handler to fail if I move the code a bit, but the functionality would still work? 
HmctsAdminNotificationHandler - I know it's not your code, probably remainder of some old logic, but it's not even handler, it just gives address from config. 

Of course the above means the dependencies need to be managed on the side of the test which is not ideal as well. I don't want you to change it here (maybe apart from ObjectMapper below), but I would like you to have an opinion and consider all pros and cons for the future :).